packages/moderation/data/nags/core.ts

```diff
@@ -0,0 +1,142 @@
+import type { Nag, NagContext } from '../../types/nags'
+import { formatProjectType } from '@modrinth/utils'
+
+export const coreNags: Nag[] = [
+  {
+    id: 'moderator-feedback',
+    title: 'Review moderator feedback',
```

Comment:
these really should use i18n for their titles and descriptions, this refactor is a good opportunity to introduce that